config-templates/000-default.conf

```diff
@@ -1,5 +1,5 @@
 <VirtualHost *:80>
-        ServerName localhost
+        ServerName alexbibbox.mupa.local
```

Comment:
Again default values?
remove those files as they should be generated during the installation? and add to git ignore?

Reply:
done